applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -216,7 +226,12 @@ validate(Context, Id, ?PREVIEW) ->
     DbId = kz_notification:db_id(Id),
     update_notification(maybe_update_db(Context), DbId);
 validate(Context, ?SMTP_LOG, Id) ->
-    load_smtp_log_doc(Id, Context).
+    load_smtp_log_doc(Id, Context);
+validate(Context, ?CUSTOMER_UPDATE, ?SEND_MESSAGE) ->
+    cb_context:set_resp_status(Context, 'success').
+
+validate(Context, ?CUSTOMER_UPDATE, ?SEND_MESSAGE, _Id) ->
```

Comment:
You need to validate that the `recipient_id` is a child of the account making the request.